boards/serpente/include/pwm_params.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Marian Buschsieweke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_serpente
+ * @{
+ *
+ * @file
+ * @brief       Configuration of SAUL mapped PWM channels
+ *
+ * @author      Marian Buschsieweke <marian.buschsieweke@ovgu.de>
+ */
+
+#ifndef PWM_PARAMS_H
+#define PWM_PARAMS_H
+
+#include "board.h"
+#include "saul/periph.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static const saul_pwm_rgb_params_t saul_pwm_rgb_params[] =
+{
+    {
+        .name  = "RGB LED",
```

Comment:
I'd rather pick "LED" as designation; that's the commonality between the LED_R, LED_G and LED_B on the pinout. (The alternative is "D3" from the schematic). It being an RGB is indicated in the SAUL type.

Reply:
I'm currntly afk (writing from phone). But I agree and will apply tonight.

Reply:
Btw: the allow edits by maintainers checkbox should be checked :-) So if you feel the itch to try that out, here is your chance ;-)